frontend/src/hooks/api/actions/useAuthSettingsApi/useAuthSettingsApi.ts

```diff
@@ -12,8 +12,9 @@ export const handleBadRequest = async (
     if (!setErrors) return;
     if (res) {
         const data = await res.json();
-        setErrors({ message: data.message });
-        throw new Error(data.message);
+        const message = data.details?.[0]?.message ?? data.message;
+        setErrors({ message });
+        throw new Error(message);
```

Comment:
I know we're keeping this behavior but it's weird that we handle setErrors and then re-throw again

Reply:
Yeah, I think it's because this one sets the error in the form (not the toast), but then we catch the second error and use that to populate the toast. But yaeh, not looking at that now 😅